### PR TITLE
[ResourceBundle] #2601 ObjectToIdentifierType is replaced by ResourceChoiceType

### PR DESCRIPTION
**Pull request by aRn0D**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | yes
| Deprecations? | yes
| Fixed tickets | #2601
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Form/Type/Rule/ContainsProductConfigurationType.php [hunk 1]
```diff
@@ -46,9 +46,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $variantRepository = $this->variantRepository;
 
         $builder
-            ->add('variant', 'sylius_entity_to_identifier', array(
-                'label'         => 'sylius.form.action.add_product_configuration.variant',
-                'class'         => $variantRepository->getClassName(),
+            ->add('variant', 'sylius_product_variant_matchr', array(
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CoreBundle/Form/Type/Rule/ContainsProductConfigurationType.php]: matchr

Comment (aRn0D) [inline src/Sylius/Bundle/CoreBundle/Form/Type/Rule/ContainsProductConfigurationType.php]: fixed thanks !

**Comment by aRn0D** [comment]
@Sylius/core-team green, any feedback ?

**Comment by aRn0D** [comment]
ping @pjedrzejewski @michalmarcinkowski

**Comment by pjedrzejewski** [comment]
I'm redoing this in my branch, cause this won't work this way. We need to get scalar id, not object. I will introduce generic resource type for this. :)

**Comment by aRn0D** [comment]
ok, plz have a look on other PR (spec ones)!

**Pull request closed without merging**